### DISCUSSION TITLE
Release Proposal for Apache Kvrocks 2.10.0

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#2559] [General]
It's been more than 3 months since the last release ([v2.9.0](https://github.com/apache/kvrocks/releases/tag/v2.9.0)), and many exciting new features and fixes have been merged into the unstable branch. Now I would like to propose the next release version 2.10.0.

All commits on the unstable branch until [b8cf118](https://github.com/apache/kvrocks/commit/b8cf118bbe56984c7112c0929402f79d81d8a0d7) (included) will be considered as commits for this release, except Kvrocks Search command(FT.*) since it's not ready to production for now.

The deadline for the cherry-picking period has been set as Oct 2, 2024, at 12:00 UTC+0. No new commits will be accepted after this time.

Feel free to leave your comments about this release proposal.

For the release process, please refer to: https://kvrocks.apache.org/community/create-a-release

@apache/kvrocks-committers

**Comment by PragmaTwice** [comment]
+1.

Commands of Kvrocks Search can be disabled in this release since it's not yet production ready.

**Reply by git-hulk** [reply]
@PragmaTwice Thank you! Will disable all FT.* commands and have updated this in the proposal.

**Comment by torwig** [comment]
+1.

**Comment by Yangsx-1** [comment]
+1.

**Comment by mapleFU** [comment]
+1

**Comment by Beihao-Zhou** [comment]
+1

**Comment by aleksraiden** [comment]
+1

**Comment by git-hulk** [comment]
Thanks to everyone who supports this release. The cherry-picking period ended, I will start the vote call soon.
